azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/fragment/calling/participantlist/ParticipantListViewModel.kt

```diff
@@ -136,7 +125,7 @@ internal class ParticipantListViewModel(
     }
 }
 
-internal data class ParticipantListViewViewModel(
+internal data class ParticipantListContent(
```

Comment:
reasoning behind the rename?

Reply:
Yes, it was brought by Adam that this name is not good.